src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -418,6 +418,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <CompileWasmArgs Condition="'$(IlcLlvmExceptionHandlingModel)' == 'cpp'">$(CompileWasmArgs) -mllvm -enable-emscripten-cxx-exceptions</CompileWasmArgs>
 
       <ScriptExt Condition="'$(OS)' == 'Windows_NT'">.bat</ScriptExt>
+      <ScriptExt Condition="'$(OS)' != 'Windows_NT'"></ScriptExt>
```

Comment:
Is `ScriptExt` set by something upstream? If so, this variable should be renamed to something less generic (e. g. `_IlcScriptExt`, or something like that).

Reply:
renamed.